CONTRIBUTING.md

```diff
@@ -43,6 +43,39 @@ When creating an issue, please ensure you include the following information
 6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
 7. When all PRs are completed the issue is then closed.
 
+## Documentation impact
+
+### documenting your change
+
+When noting there is a doc impact on your change, you have two choices:
+
+1. Submit an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues)
+OR
+2. Contribute your own doc change to [docs-rpc](https://github.com/rackerlabs/docs-rpc) and tag a relevant docs person for review and merge.
+
+### what to document
+
+Document changes if:
+
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+### how to document
+
+The documentation work is all done in [this](https://github.com/rackerlabs/docs-rpc) repo.
+
+1. See the [Docs workflow](https://github.com/rackerlabs/docs-rpc/blob/master/GITHUBING.rst) to setup for contribution and
+   [here](https://github.com/rackerlabs/docs-rpc/blob/master/CONTRIBUTING.rst) for our Contributor guidelines.
+2. Make changes in own branch off fork.
+3. Write relevant commit message.
+4. Push up to docs-rpc.
```

Comment:
Is step 4 the create pull request step?

Steps 2, 3 and 4 seem like an inconsistent level of detail and they are already covered in the links in step 1.

Maybe remove 2-4 and amend step 5 by prepending it with 'Once you have created a pull request, '.